charts/rancher-eks-operator/102.0.1+up1.1.5/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+annotations:
+  catalog.cattle.io/auto-install: rancher-eks-operator-crd=match
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/hidden: "true"
```

Comment:
This chart is supposed to have this annotation - `catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'` so it's deployed on 2.7.0 and 2.7.1

Reply:
The one that is failing, `[charts/rancher-eks-operator/102.0.1+up1.1.5/Chart.yaml](https://github.com/rancher/charts/pull/2538/files/1486ee5e6387c9bdeae107212675d3b2945efac3#diff-7190bcffbb7ee960b29aefc04f91dc0340f63a89a04575706310b8634e9543b5)`

Reply:
The PR should not be making edits to an already existing chart. I was under the impression that was a common knowledge thing. What should be done is to release a new chart with a minor version bump and to hide the old chart.

Reply:
ok, it is done

Reply:
@sowmyav27 I don't think that is correct. Both new charts will need the annotation `catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0` as originally requested because they do not function properly on those rancher versions.

@eliyamlevy Same here. That is one of the to-be-released versions, not the bad chart.